### DISCUSSION TITLE
Blog Posts for Deploying Concourse on GKE

### DISCUSSION THREAD

**Discussion by cunnie** [concourse/concourse#7516] [Show and tell]
Hi everyone,

If you want to set up a public-facing Concourse on Kubernetes & want an opinionated way to do it, I wrote a series of blog posts to accomplish that:

1. [Terraform](https://blog.nono.io/post/concourse_on_k8s-1/)
2. [Ingress](https://blog.nono.io/post/concourse_on_k8s-2/)
3. [TLS](https://blog.nono.io/post/concourse_on_k8s-3/)
4. [Concourse itself](https://blog.nono.io/post/concourse_on_k8s-4/)

Any & all feedback is welcome. Thanks, Brian

**Comment by ringods** [comment]
In case of AWS, look here: https://github.com/skyscrapers/terraform-concourse

**Reply by cunnie** [reply]
Thanks @ringods , I've updated my blog post to include a link to that repo: <https://blog.nono.io/post/concourse_on_k8s-1/#references>

**Comment by cunnie** [comment]
Now including a [new post](https://blog.nono.io/post/concourse_on_k8s-5/) for Vault <-> Concourse integration (using `app_role`).
